python/paddle/v2/framework/tests/test_layers.py

```diff
@@ -18,8 +19,8 @@ def test_fit_a_line(self):
 
         avg_cost = layers.mean(x=cost, program=program)
         self.assertIsNotNone(avg_cost)
-        program.append_backward(avg_cost, set())
-        print str(program)
+        program.append_backward(avg_cost)
+        # print str(program)
```

Comment:
clean up.

Reply:
Actually, this line should not be committed. It is committed now to reduce test output.